### ISSUE TITLE
Request : filter for permanently delete (shift+del)

### ISSUE THREAD

**Issue by winproof** [RealRaven2000/quickFilters#64]
hello!
very good tool, just one filter is missing, a filter for permanently delete an email, without push it on trash.
Thanks!

**Comment by RealRaven2000** [comment]
For adding new filter actions / conditions the FiltaQuilla Add-on is a better fit:

https://github.com/RealRaven2000/FiltaQuilla/issues

Once you have added the new actions there they can actually be added to a quickFilter custom filter template via the qF Advanced options:
![image](https://user-images.githubusercontent.com/6314674/121010996-02220700-c78e-11eb-97ca-d073a9b543a9.png)

**Comment by winproof** [comment]
sorry, i don' t really understand your answer (english is not my spoken language, maybe i miss something 😁 )
your right, i've just try quickfilter, it's better for adding filter, but it's not solve my problem, there is not action "permanently delete".
how i can do that?
do i need to create a second rule for deleting email from trash?

**Comment by RealRaven2000** [comment]
> sorry, i don' t really understand your answer (english is not my spoken language, maybe i miss something  )
> your right, i've just try quickfilter, it's better for adding filter, but it's not solve my problem, there is not action "permanently delete".

yes, quickFilters cannot add actions that **do not exist in Thunderbird**. There is another Add-on FiltaQuilla that does this (add other actions and search fields). But I don't know that FiltaQuilla currently has a "delete permanently" action. So if you wanted this added as  a new action type, you should probably request it on the [FiltaQuilla github](https://github.com/RealRaven2000/FiltaQuilla/issues).

**Comment by winproof** [comment]
ok, my fault, sorry, I thought I was on the filtaquilla github, I posted in the wrong place, I had not seen that I was in fact on the quickfilters github :)
